README.md

```diff
@@ -31,14 +31,14 @@ Details of the repository structure and contribution guidelines are listed on th
 Here is a quick starter summary for the most common use case.
 
 **Adding an ERC20 token checklist**:
-- [ ] Make sure your smartcontract has more than 2,500 address holders, otherwise you will be rejected
-- [ ] Fork the Github repository
-- [ ] Create folder with name of token smartcontact address in [_checksum format_](https://developer.trustwallet.com/add_new_asset#checksum_format) `blockchains/ethereum/assets/<token_smartcontract_address>/`.
-- [ ] Tell your designer that token image must be in PNG format, avoid transparent background, recommended size 256x256px, max. 512x512px, with max file size of 100kB, for further details read [image rules](https://developer.trustwallet.com/add_new_asset#image-requirements).
-- [ ] Upload your logo with file named `logo.png` to previously created folder with smartcontract address, and if you done all correctly your path should look like this. `blockchains/ethereum/assets/0x1234567461d3f8Db7496581774Bd869C83D51c93/logo.png`
-- [ ] Create `info.json` file with info about the token/project
-- [ ] Create a pull request to the main repo
-- [ ] Pay the processing fee
+- [☑️] Make sure your smartcontract has more than 2,500 address holders, otherwise you will be rejected
+- [☑️] Fork the Github repository
```

Comment:
2022-02-17T12:32:32.8225582Z Requested labels: ubuntu-latest
2022-02-17T12:32:32.8225637Z Job defined at: alexanderramos89/asset/.github/workflows/fix.yml@refs/heads/alexanderramos89-patch-5
2022-02-17T12:32:32.8225660Z Waiting for a runner to pick up this job...
2022-02-17T12:32:33.2797029Z Job is waiting for a hosted runner to come online.
2022-02-17T12:32:36.5771751Z Job is about to start running on the hosted runner: Hosted Agent (hosted)
2022-02-17T12:32:40.1394007Z Current runner version: '2.287.1'
2022-02-17T12:32:40.1419325Z ##[group]Operating System
2022-02-17T12:32:40.1419837Z Ubuntu
2022-02-17T12:32:40.1420061Z 20.04.3
2022-02-17T12:32:40.1420308Z LTS
2022-02-17T12:32:40.1420531Z ##[endgroup]
2022-02-17T12:32:40.1420865Z ##[group]Virtual Environment
2022-02-17T12:32:40.1421169Z Environment: ubuntu-20.04
2022-02-17T12:32:40.1421490Z Version: 20220213.1
2022-02-17T12:32:40.1421973Z Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20220213.1/images/linux/Ubuntu2004-Readme.md
2022-02-17T12:32:40.1422537Z Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20220213.1
2022-02-17T12:32:40.1422973Z ##[endgroup]
2022-02-17T12:32:40.1423272Z ##[group]Virtual Environment Provisioner
2022-02-17T12:32:40.1423602Z 1.0.0.0-main-20220202-1
2022-02-17T12:32:40.1423879Z ##[endgroup]
2022-02-17T12:32:40.1424707Z ##[group]GITHUB_TOKEN Permissions
2022-02-17T12:32:40.1425294Z Actions: write
2022-02-17T12:32:40.1425672Z Checks: write
2022-02-17T12:32:40.1426012Z Contents: write
2022-02-17T12:32:40.1426262Z Deployments: write
2022-02-17T12:32:40.1426604Z Discussions: write
2022-02-17T12:32:40.1426890Z Issues: write
2022-02-17T12:32:40.1427125Z Metadata: read
2022-02-17T12:32:40.1427404Z Packages: write
2022-02-17T12:32:40.1427638Z Pages: write
2022-02-17T12:32:40.1427960Z PullRequests: write
2022-02-17T12:32:40.1428236Z RepositoryProjects: write
2022-02-17T12:32:40.1428554Z SecurityEvents: write
2022-02-17T12:32:40.1428850Z Statuses: write
2022-02-17T12:32:40.1429101Z ##[endgroup]
2022-02-17T12:32:40.1432680Z Secret source: Actions
2022-02-17T12:32:40.1433076Z Prepare workflow directory
2022-02-17T12:32:40.2283354Z Prepare all required actions
2022-02-17T12:32:40.2471408Z Getting action download info
2022-02-17T12:32:40.4592853Z Download action repository 'actions/checkout@v2' (SHA:ec3a7ce113134d7a93b817d10a8272cb61118579)
2022-02-17T12:32:41.1077669Z Download action repository 'actions/setup-node@v1' (SHA:f1f314fca9dfce2769ece7d933488f076716723e)
2022-02-17T12:32:41.6148444Z Download action repository 'stefanzweifel/git-auto-commit-action@v4.1.2' (SHA:dd055f6225757c369f948744463f63ac1bd4c277)
2022-02-17T12:32:42.1600107Z ##[group]Run actions/checkout@v2
2022-02-17T12:32:42.1600447Z with:
2022-02-17T12:32:42.1601255Z   token: ***
2022-02-17T12:32:42.1601521Z   ref: refs/heads/alexanderramos89-patch-5
2022-02-17T12:32:42.1601866Z   repository: alexanderramos89/asset
2022-02-17T12:32:42.1602117Z   ssh-strict: true
2022-02-17T12:32:42.1602395Z   persist-credentials: true
2022-02-17T12:32:42.1602628Z   clean: true
2022-02-17T12:32:42.1602919Z   fetch-depth: 1
2022-02-17T12:32:42.1603188Z   lfs: false
2022-02-17T12:32:42.1603393Z   submodules: false
2022-02-17T12:32:42.1603654Z ##[endgroup]
2022-02-17T12:32:42.3918619Z Syncing repository: alexanderramos89/asset
2022-02-17T12:32:42.3920307Z ##[group]Getting Git version info
2022-02-17T12:32:42.3920807Z Working directory is '/home/runner/work/asset/asset'
2022-02-17T12:32:42.3921263Z [command]/usr/bin/git version
2022-02-17T12:32:42.4017798Z git version 2.35.1
2022-02-17T12:32:42.4042552Z ##[endgroup]
2022-02-17T12:32:42.4051271Z Deleting the contents of '/home/runner/work/asset/asset'
2022-02-17T12:32:42.4058521Z ##[group]Initializing the repository
2022-02-17T12:32:42.4064075Z [command]/usr/bin/git init /home/runner/work/asset/asset
2022-02-17T12:32:42.4147254Z hint: Using 'master' as the name for the initial branch. This default branch name
2022-02-17T12:32:42.4148011Z hint: is subject to change. To configure the initial branch name to use in all
2022-02-17T12:32:42.4148920Z hint: of your new repositories, which will suppress this warning, call:
2022-02-17T12:32:42.4149342Z hint: 
2022-02-17T12:32:42.4149916Z hint: 	git config --global init.defaultBranch <name>
2022-02-17T12:32:42.4150565Z hint: 
2022-02-17T12:32:42.4151131Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2022-02-17T12:32:42.4151603Z hint: 'development'. The just-created branch can be renamed via this command:
2022-02-17T12:32:42.4152069Z hint: 
2022-02-17T12:32:42.4152307Z hint: 	git branch -m <name>
2022-02-17T12:32:42.4170341Z Initialized empty Git repository in /home/runner/work/asset/asset/.git/
2022-02-17T12:32:42.4181548Z [command]/usr/bin/git remote add origin https://github.com/alexanderramos89/asset
2022-02-17T12:32:42.4232226Z ##[endgroup]
2022-02-17T12:32:42.4232659Z ##[group]Disabling automatic garbage collection
2022-02-17T12:32:42.4234803Z [command]/usr/bin/git config --local gc.auto 0
2022-02-17T12:32:42.4264169Z ##[endgroup]
2022-02-17T12:32:42.4283385Z ##[group]Setting up auth
2022-02-17T12:32:42.4284136Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2022-02-17T12:32:42.4306518Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2022-02-17T12:32:42.4634905Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2022-02-17T12:32:42.4683819Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2022-02-17T12:32:42.4895187Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2022-02-17T12:32:42.4933596Z ##[endgroup]
2022-02-17T12:32:42.4934031Z ##[group]Fetching the repository
2022-02-17T12:32:42.4940006Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +refs/heads/alexanderramos89-patch-5:refs/remotes/origin/alexanderramos89-patch-5
2022-02-17T12:32:42.9799926Z remote: Enumerating objects: 30326, done.        
2022-02-17T12:32:42.9815770Z remote: Counting objects:   0% (1/30326)        
2022-02-17T12:32:42.9818624Z remote: Counting objects:   1% (304/30326)        
2022-02-17T12:32:42.9819398Z remote: Counting objects:   2% (607/30326)        
2022-02-17T12:32:42.9820972Z remote: Counting objects:   3% (910/30326)        
2022-02-17T12:32:42.9823930Z remote: Counting objects:   4% (1214/30326)        
2022-02-17T12:32:42.9824445Z remote: Counting objects:   5% (1517/30326)        
2022-02-17T12:32:42.9826655Z remote: Counting objects:   6% (1820/30326)        
2022-02-17T12:32:42.9827505Z remote: Counting objects:   7% (2123/30326)        
2022-02-17T12:32:42.9829479Z remote: Counting objects:   8% (2427/30326)        
2022-02-17T12:32:42.9829908Z remote: Counting objects:   9% (2730/30326)        
2022-02-17T12:32:42.9830455Z remote: Counting objects:  10% (3033/30326)        
2022-02-17T12:32:43.0210255Z remote: Counting objects:  11% (3336/30326)        
2022-02-17T12:32:43.0210923Z remote: Counting objects:  12% (3640/30326)        
2022-02-17T12:32:43.0211421Z remote: Counting objects:  13% (3943/30326)        
2022-02-17T12:32:43.0212028Z remote: Counting objects:  14% (4246/30326)        
2022-02-17T12:32:43.0212479Z remote: Counting objects:  15% (4549/30326)        
2022-02-17T12:32:43.0213015Z remote: Counting objects:  16% (4853/30326)        
2022-02-17T12:32:43.0213500Z remote: Counting objects:  17% (5156/30326)        
2022-02-17T12:32:43.0214043Z remote: Counting objects:  18% (5459/30326)        
2022-02-17T12:32:43.0214499Z remote: Counting objects:  19% (5762/30326)        
2022-02-17T12:32:43.0214966Z remote: Counting objects:  20% (6066/30326)        
2022-02-17T12:32:43.0215504Z remote: Counting objects:  21% (6369/30326)        
2022-02-17T12:32:43.0215935Z remote: Counting objects:  22% (6672/30326)        
2022-02-17T12:32:43.0216406Z remote: Counting objects:  23% (6975/30326)        
2022-02-17T12:32:43.0216859Z remote: Counting objects:  24% (7279/30326)        
2022-02-17T12:32:43.0217267Z remote: Counting objects:  25% (7582/30326)        
2022-02-17T12:32:43.0217628Z remote: Counting objects:  26% (7885/30326)        
2022-02-17T12:32:43.0218102Z remote: Counting objects:  27% (8189/30326)        
2022-02-17T12:32:43.0219504Z remote: Counting objects:  28% (8492/30326)        
2022-02-17T12:32:43.0219831Z remote: Counting objects:  29% (8795/30326)        
2022-02-17T12:32:43.0220155Z remote: Counting objects:  30% (9098/30326)        
2022-02-17T12:32:43.0220453Z remote: Counting objects:  31% (9402/30326)        
2022-02-17T12:32:43.0220757Z remote: Counting objects:  32% (9705/30326)        
2022-02-17T12:32:43.0221069Z remote: Counting objects:  33% (10008/30326)        
2022-02-17T12:32:43.0221397Z remote: Counting objects:  34% (10311/30326)        
2022-02-17T12:32:43.0221731Z remote: Counting objects:  35% (10615/30326)        
2022-02-17T12:32:43.0222033Z remote: Counting objects:  36% (10918/30326)        
2022-02-17T12:32:43.0222328Z remote: Counting objects:  37% (11221/30326)        
2022-02-17T12:32:43.0222642Z remote: Counting objects:  38% (11524/30326)        
2022-02-17T12:32:43.0222935Z remote: Counting objects:  39% (11828/30326)        
2022-02-17T12:32:43.0223254Z remote: Counting objects:  40% (12131/30326)        
2022-02-17T12:32:43.0223557Z remote: Counting objects:  41% (12434/30326)        
2022-02-17T12:32:43.0223861Z remote: Counting objects:  42% (12737/30326)        
2022-02-17T12:32:43.0224156Z remote: Counting objects:  43% (13041/30326)        
2022-02-17T12:32:43.0224477Z remote: Counting objects:  44% (13344/30326)        
2022-02-17T12:32:43.0224774Z remote: Counting objects:  45% (13647/30326)        
2022-02-17T12:32:43.0225071Z remote: Counting objects:  46% (13950/30326)        
2022-02-17T12:32:43.0225380Z remote: Counting objects:  47% (14254/30326)        
2022-02-17T12:32:43.0225672Z remote: Counting objects:  48% (14557/30326)        
2022-02-17T12:32:43.0225960Z remote: Counting objects:  49% (14860/30326)        
2022-02-17T12:32:43.0226283Z remote: Counting objects:  50% (15163/30326)        
2022-02-17T12:32:43.0226583Z remote: Counting objects:  51% (15467/30326)        
2022-02-17T12:32:43.0226897Z remote: Counting objects:  52% (15770/30326)        
2022-02-17T12:32:43.0227216Z remote: Counting objects:  53% (16073/30326)        
2022-02-17T12:32:43.0227535Z remote: Counting objects:  54% (16377/30326)        
2022-02-17T12:32:43.0227965Z remote: Counting objects:  55% (16680/30326)        
2022-02-17T12:32:43.0228370Z remote: Counting objects:  56% (16983/30326)        
2022-02-17T12:32:43.0228692Z remote: Counting objects:  57% (17286/30326)        
2022-02-17T12:32:43.0229007Z remote: Counting objects:  58% (17590/30326)        
2022-02-17T12:32:43.0229339Z remote: Counting objects:  59% (17893/30326)        
2022-02-17T12:32:43.0229650Z remote: Counting objects:  60% (18196/30326)        
2022-02-17T12:32:43.0230025Z remote: Counting objects:  61% (18499/30326)        
2022-02-17T12:32:43.0230456Z remote: Counting objects:  62% (18803/30326)        
2022-02-17T12:32:43.0230908Z remote: Counting objects:  63% (19106/30326)        
2022-02-17T12:32:43.0231357Z remote: Counting objects:  64% (19409/30326)        
2022-02-17T12:32:43.0231792Z remote: Counting objects:  65% (19712/30326)        
2022-02-17T12:32:43.0232209Z remote: Counting objects:  66% (20016/30326)        
2022-02-17T12:32:43.0232609Z remote: Counting objects:  67% (20319/30326)        
2022-02-17T12:32:43.0233017Z remote: Counting objects:  68% (20622/30326)        
2022-02-17T12:32:43.0233440Z remote: Counting objects:  69% (20925/30326)        
2022-02-17T12:32:43.0233821Z remote: Counting objects:  70% (21229/30326)        
2022-02-17T12:32:43.0234229Z remote: Counting objects:  71% (21532/30326)        
2022-02-17T12:32:43.0234643Z remote: Counting objects:  72% (21835/30326)        
2022-02-17T12:32:43.0235054Z remote: Counting objects:  73% (22138/30326)        
2022-02-17T12:32:43.0235447Z remote: Counting objects:  74% (22442/30326)        
2022-02-17T12:32:43.0235862Z remote: Counting objects:  75% (22745/30326)        
2022-02-17T12:32:43.0236281Z remote: Counting objects:  76% (23048/30326)        
2022-02-17T12:32:43.0236785Z remote: Counting objects:  77% (23352/30326)        
2022-02-17T12:32:43.0237278Z remote: Counting objects:  78% (23655/30326)        
2022-02-17T12:32:43.0237606Z remote: Counting objects:  79% (23958/30326)        
2022-02-17T12:32:43.0237926Z remote: Counting objects:  80% (24261/30326)        
2022-02-17T12:32:43.0238276Z remote: Counting objects:  81% (24565/30326)        
2022-02-17T12:32:43.0238591Z remote: Counting objects:  82% (24868/30326)        
2022-02-17T12:32:43.0239027Z remote: Counting objects:  83% (25171/30326)        
2022-02-17T12:32:43.0239432Z remote: Counting objects:  84% (25474/30326)        
2022-02-17T12:32:43.0239845Z remote: Counting objects:  85% (25778/30326)        
2022-02-17T12:32:43.0240234Z remote: Counting objects:  86% (26081/30326)        
2022-02-17T12:32:43.0240644Z remote: Counting objects:  87% (26384/30326)        
2022-02-17T12:32:43.0241043Z remote: Counting objects:  88% (26687/30326)        
2022-02-17T12:32:43.0241454Z remote: Counting objects:  89% (26991/30326)        
2022-02-17T12:32:43.0241855Z remote: Counting objects:  90% (27294/30326)        
2022-02-17T12:32:43.0242264Z remote: Counting objects:  91% (27597/30326)        
2022-02-17T12:32:43.0242667Z remote: Counting objects:  92% (27900/30326)        
2022-02-17T12:32:43.0243074Z remote: Counting objects:  93% (28204/30326)        
2022-02-17T12:32:43.0243462Z remote: Counting objects:  94% (28507/30326)        
2022-02-17T12:32:43.0243899Z remote: Counting objects:  95% (28810/30326)        
2022-02-17T12:32:43.0244306Z remote: Counting objects:  96% (29113/30326)        
2022-02-17T12:32:43.0244714Z remote: Counting objects:  97% (29417/30326)        
2022-02-17T12:32:43.0245109Z remote: Counting objects:  98% (29720/30326)        
2022-02-17T12:32:43.0245537Z remote: Counting objects:  99% (30023/30326)        
2022-02-17T12:32:43.0245943Z remote: Counting objects: 100% (30326/30326)        
2022-02-17T12:32:43.0246263Z remote: Counting objects: 100% (30326/30326), done.        
2022-02-17T12:32:43.0246783Z remote: Compressing objects:   0% (1/26453)        
2022-02-17T12:32:43.0265688Z remote: Compressing objects:   1% (265/26453)        
2022-02-17T12:32:43.0350784Z remote: Compressing objects:   2% (530/26453)        
2022-02-17T12:32:43.0423328Z remote: Compressing objects:   3% (794/26453)        
2022-02-17T12:32:43.0496390Z remote: Compressing objects:   4% (1059/26453)        
2022-02-17T12:32:43.0583407Z remote: Compressing objects:   5% (1323/26453)        
2022-02-17T12:32:43.0660171Z remote: Compressing objects:   6% (1588/26453)        
2022-02-17T12:32:43.0741050Z remote: Compressing objects:   7% (1852/26453)        
2022-02-17T12:32:43.0812113Z remote: Compressing objects:   8% (2117/26453)        
2022-02-17T12:32:43.0898061Z remote: Compressing objects:   9% (2381/26453)        
2022-02-17T12:32:43.0965688Z remote: Compressing objects:  10% (2646/26453)        
2022-02-17T12:32:43.1066457Z remote: Compressing objects:  11% (2910/26453)        
2022-02-17T12:32:43.1111650Z remote: Compressing objects:  12% (3175/26453)        
2022-02-17T12:32:43.1195109Z remote: Compressing objects:  13% (3439/26453)        
2022-02-17T12:32:43.1278595Z remote: Compressing objects:  14% (3704/26453)        
2022-02-17T12:32:43.1362800Z remote: Compressing objects:  15% (3968/26453)        
2022-02-17T12:32:43.1422507Z remote: Compressing objects:  16% (4233/26453)        
2022-02-17T12:32:43.1501426Z remote: Compressing objects:  17% (4498/26453)        
2022-02-17T12:32:43.1580973Z remote: Compressing objects:  18% (4762/26453)        
2022-02-17T12:32:43.1647253Z remote: Compressing objects:  19% (5027/26453)        
2022-02-17T12:32:43.1740541Z remote: Compressing objects:  20% (5291/26453)        
2022-02-17T12:32:43.1817552Z remote: Compressing objects:  21% (5556/26453)        
2022-02-17T12:32:43.1871642Z remote: Compressing objects:  22% (5820/26453)        
2022-02-17T12:32:43.1936108Z remote: Compressing objects:  23% (6085/26453)        
2022-02-17T12:32:43.2009189Z remote: Compressing objects:  24% (6349/26453)        
2022-02-17T12:32:43.2076470Z remote: Compressing objects:  25% (6614/26453)        
2022-02-17T12:32:43.2150505Z remote: Compressing objects:  26% (6878/26453)        
2022-02-17T12:32:43.2217579Z remote: Compressing objects:  27% (7143/26453)        
2022-02-17T12:32:43.2289791Z remote: Compressing objects:  28% (7407/26453)        
2022-02-17T12:32:43.2365492Z remote: Compressing objects:  29% (7672/26453)        
2022-02-17T12:32:43.2432636Z remote: Compressing objects:  30% (7936/26453)        
2022-02-17T12:32:43.2496936Z remote: Compressing objects:  31% (8201/26453)        
2022-02-17T12:32:43.2497343Z remote: Compressing objects:  32% (8465/26453)        
2022-02-17T12:32:43.2503918Z remote: Compressing objects:  33% (8730/26453)        
2022-02-17T12:32:43.2504303Z remote: Compressing objects:  34% (8995/26453)        
2022-02-17T12:32:43.2504821Z remote: Compressing objects:  35% (9259/26453)        
2022-02-17T12:32:43.2514219Z remote: Compressing objects:  36% (9524/26453)        
2022-02-17T12:32:43.2537481Z remote: Compressing objects:  37% (9788/26453)        
2022-02-17T12:32:43.2537914Z remote: Compressing objects:  38% (10053/26453)        
2022-02-17T12:32:43.2571693Z remote: Compressing objects:  39% (10317/26453)        
2022-02-17T12:32:43.2577142Z remote: Compressing objects:  40% (10582/26453)        
2022-02-17T12:32:43.2577534Z remote: Compressing objects:  41% (10846/26453)        
2022-02-17T12:32:43.2603286Z remote: Compressing objects:  42% (11111/26453)        
2022-02-17T12:32:43.2603698Z remote: Compressing objects:  43% (11375/26453)        
2022-02-17T12:32:43.2668837Z remote: Compressing objects:  44% (11640/26453)        
2022-02-17T12:32:43.2669263Z remote: Compressing objects:  45% (11904/26453)        
2022-02-17T12:32:43.2670574Z remote: Compressing objects:  46% (12169/26453)        
2022-02-17T12:32:43.2731515Z remote: Compressing objects:  47% (12433/26453)        
2022-02-17T12:32:43.2732171Z remote: Compressing objects:  48% (12698/26453)        
2022-02-17T12:32:43.2733313Z remote: Compressing objects:  49% (12962/26453)        
2022-02-17T12:32:43.2736675Z remote: Compressing objects:  50% (13227/26453)        
2022-02-17T12:32:43.2737292Z remote: Compressing objects:  51% (13492/26453)        
2022-02-17T12:32:43.2737640Z remote: Compressing objects:  52% (13756/26453)        
2022-02-17T12:32:43.2738536Z remote: Compressing objects:  53% (14021/26453)        
2022-02-17T12:32:43.2738964Z remote: Compressing objects:  54% (14285/26453)        
2022-02-17T12:32:43.2739744Z remote: Compressing objects:  55% (14550/26453)        
2022-02-17T12:32:43.2740340Z remote: Compressing objects:  56% (14814/26453)        
2022-02-17T12:32:43.2741038Z remote: Compressing objects:  57% (15079/26453)        
2022-02-17T12:32:43.2741683Z remote: Compressing objects:  58% (15343/26453)        
2022-02-17T12:32:43.2742357Z remote: Compressing objects:  59% (15608/26453)        
2022-02-17T12:32:43.2743013Z remote: Compressing objects:  60% (15872/26453)        
2022-02-17T12:32:43.2780532Z remote: Compressing objects:  61% (16137/26453)        
2022-02-17T12:32:43.2787211Z remote: Compressing objects:  62% (16401/26453)        
2022-02-17T12:32:43.2801467Z remote: Compressing objects:  63% (16666/26453)        
2022-02-17T12:32:43.2831498Z remote: Compressing objects:  64% (16930/26453)        
2022-02-17T12:32:43.2831849Z remote: Compressing objects:  65% (17195/26453)        
2022-02-17T12:32:43.2851274Z remote: Compressing objects:  66% (17459/26453)        
2022-02-17T12:32:43.2851789Z remote: Compressing objects:  67% (17724/26453)        
2022-02-17T12:32:43.2855373Z remote: Compressing objects:  68% (17989/26453)        
2022-02-17T12:32:43.2855888Z remote: Compressing objects:  69% (18253/26453)        
2022-02-17T12:32:43.2902923Z remote: Compressing objects:  70% (18518/26453)        
2022-02-17T12:32:43.2903781Z remote: Compressing objects:  71% (18782/26453)        
2022-02-17T12:32:43.2904086Z remote: Compressing objects:  72% (19047/26453)        
2022-02-17T12:32:43.2904527Z remote: Compressing objects:  73% (19311/26453)        
2022-02-17T12:32:43.2905069Z remote: Compressing objects:  74% (19576/26453)        
2022-02-17T12:32:43.2905704Z remote: Compressing objects:  75% (19840/26453)        
2022-02-17T12:32:43.2906404Z remote: Compressing objects:  76% (20105/26453)        
2022-02-17T12:32:43.2907003Z remote: Compressing objects:  77% (20369/26453)        
2022-02-17T12:32:43.2907722Z remote: Compressing objects:  78% (20634/26453)        
2022-02-17T12:32:43.2908847Z remote: Compressing objects:  79% (20898/26453)        
2022-02-17T12:32:43.2909330Z remote: Compressing objects:  80% (21163/26453)        
2022-02-17T12:32:43.2910056Z remote: Compressing objects:  81% (21427/26453)        
2022-02-17T12:32:43.2910730Z remote: Compressing objects:  82% (21692/26453)        
2022-02-17T12:32:43.2911322Z remote: Compressing objects:  83% (21956/26453)        
2022-02-17T12:32:43.2912048Z remote: Compressing objects:  84% (22221/26453)        
2022-02-17T12:32:43.2912698Z remote: Compressing objects:  85% (22486/26453)        
2022-02-17T12:32:43.2913409Z remote: Compressing objects:  86% (22750/26453)        
2022-02-17T12:32:43.2914047Z remote: Compressing objects:  87% (23015/26453)        
2022-02-17T12:32:43.2914942Z remote: Compressing objects:  88% (23279/26453)        
2022-02-17T12:32:43.2915216Z remote: Compressing objects:  89% (23544/26453)        
2022-02-17T12:32:43.2915724Z remote: Compressing objects:  90% (23808/26453)        
2022-02-17T12:32:43.2916377Z remote: Compressing objects:  91% (24073/26453)        
2022-02-17T12:32:43.2917040Z remote: Compressing objects:  92% (24337/26453)        
2022-02-17T12:32:43.2917664Z remote: Compressing objects:  93% (24602/26453)        
2022-02-17T12:32:43.2918456Z remote: Compressing objects:  94% (24866/26453)        
2022-02-17T12:32:43.2919205Z remote: Compressing objects:  95% (25131/26453)        
2022-02-17T12:32:43.2919675Z remote: Compressing objects:  96% (25395/26453)        
2022-02-17T12:32:43.2920305Z remote: Compressing objects:  97% (25660/26453)        
2022-02-17T12:32:43.2920739Z remote: Compressing objects:  98% (25924/26453)        
2022-02-17T12:32:43.2921517Z remote: Compressing objects:  99% (26189/26453)        
2022-02-17T12:32:43.2921983Z remote: Compressing objects: 100% (26453/26453)        
2022-02-17T12:32:43.2922625Z remote: Compressing objects: 100% (26453/26453), done.        
2022-02-17T12:32:43.5093458Z Receiving objects:   0% (1/30326)
2022-02-17T12:32:43.5461707Z Receiving objects:   1% (304/30326)
2022-02-17T12:32:43.5861257Z Receiving objects:   2% (607/30326)
2022-02-17T12:32:43.6405002Z Receiving objects:   3% (910/30326)
2022-02-17T12:32:43.6679171Z Receiving objects:   4% (1214/30326)
2022-02-17T12:32:43.6960961Z Receiving objects:   5% (1517/30326)
2022-02-17T12:32:43.7259316Z Receiving objects:   6% (1820/30326)
2022-02-17T12:32:43.7575261Z Receiving objects:   7% (2123/30326)
2022-02-17T12:32:43.7999170Z Receiving objects:   8% (2427/30326)
2022-02-17T12:32:43.8469766Z Receiving objects:   9% (2730/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:43.8757071Z Receiving objects:  10% (3033/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:43.9031322Z Receiving objects:  11% (3336/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:43.9342971Z Receiving objects:  12% (3640/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:43.9748983Z Receiving objects:  13% (3943/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:44.0241333Z Receiving objects:  14% (4246/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:44.0561507Z Receiving objects:  15% (4549/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:44.0855934Z Receiving objects:  16% (4853/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:44.1163508Z Receiving objects:  17% (5156/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:44.1454955Z Receiving objects:  18% (5459/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:44.1796872Z Receiving objects:  19% (5762/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:44.2099769Z Receiving objects:  20% (6066/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:44.2447737Z Receiving objects:  21% (6369/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:44.2764339Z Receiving objects:  22% (6672/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:44.2942631Z Receiving objects:  23% (6975/30326), 16.44 MiB | 32.87 MiB/s
2022-02-17T12:32:44.3067125Z Receiving objects:  23% (7140/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.3390450Z Receiving objects:  24% (7279/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.3708993Z Receiving objects:  25% (7582/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.3982463Z Receiving objects:  26% (7885/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.4329292Z Receiving objects:  27% (8189/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.4653138Z Receiving objects:  28% (8492/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.4951425Z Receiving objects:  29% (8795/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.5297364Z Receiving objects:  30% (9098/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.5620426Z Receiving objects:  31% (9402/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.5930401Z Receiving objects:  32% (9705/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.6266805Z Receiving objects:  33% (10008/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.6639088Z Receiving objects:  34% (10311/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.6896347Z Receiving objects:  35% (10615/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.7258301Z Receiving objects:  36% (10918/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.7562258Z Receiving objects:  37% (11221/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.7852168Z Receiving objects:  38% (11524/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.8173795Z Receiving objects:  39% (11828/30326), 44.02 MiB | 44.01 MiB/s
2022-02-17T12:32:44.8656201Z Receiving objects:  40% (12131/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:44.9104230Z Receiving objects:  41% (12434/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:44.9493864Z Receiving objects:  42% (12737/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:44.9774552Z Receiving objects:  43% (13041/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:45.0115364Z Receiving objects:  44% (13344/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:45.0459716Z Receiving objects:  45% (13647/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:45.0780711Z Receiving objects:  46% (13950/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:45.1067840Z Receiving objects:  47% (14254/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:45.1406605Z Receiving objects:  48% (14557/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:45.1727293Z Receiving objects:  49% (14860/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:45.2027005Z Receiving objects:  50% (15163/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:45.2386805Z Receiving objects:  51% (15467/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:45.2702091Z Receiving objects:  52% (15770/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:45.2943663Z Receiving objects:  53% (16073/30326), 73.26 MiB | 48.83 MiB/s
2022-02-17T12:32:45.3045373Z Receiving objects:  53% (16287/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.3362398Z Receiving objects:  54% (16377/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.3680798Z Receiving objects:  55% (16680/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.4041565Z Receiving objects:  56% (16983/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.4371881Z Receiving objects:  57% (17286/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.4707806Z Receiving objects:  58% (17590/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.5014513Z Receiving objects:  59% (17893/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.5333618Z Receiving objects:  60% (18196/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.5625880Z Receiving objects:  61% (18499/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.5931565Z Receiving objects:  62% (18803/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.6253784Z Receiving objects:  63% (19106/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.6597228Z Receiving objects:  64% (19409/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.6894074Z Receiving objects:  65% (19712/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.7202173Z Receiving objects:  66% (20016/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.7566013Z Receiving objects:  67% (20319/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.7868869Z Receiving objects:  68% (20622/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.8334135Z Receiving objects:  69% (20925/30326), 101.22 MiB | 50.61 MiB/s
2022-02-17T12:32:45.8810928Z Receiving objects:  70% (21229/30326), 130.32 MiB | 52.13 MiB/s
2022-02-17T12:32:45.9280605Z Receiving objects:  71% (21532/30326), 130.32 MiB | 52.13 MiB/s
2022-02-17T12:32:45.9733710Z Receiving objects:  72% (21835/30326), 130.32 MiB | 52.13 MiB/s
2022-02-17T12:32:46.0222070Z Receiving objects:  73% (22138/30326), 130.32 MiB | 52.13 MiB/s
2022-02-17T12:32:46.0709381Z Receiving objects:  74% (22442/30326), 130.32 MiB | 52.13 MiB/s
2022-02-17T12:32:46.1222923Z Receiving objects:  75% (22745/30326), 130.32 MiB | 52.13 MiB/s
2022-02-17T12:32:46.1790357Z Receiving objects:  76% (23048/30326), 130.32 MiB | 52.13 MiB/s
2022-02-17T12:32:46.2308784Z Receiving objects:  77% (23352/30326), 130.32 MiB | 52.13 MiB/s
2022-02-17T12:32:46.2776833Z Receiving objects:  78% (23655/30326), 130.32 MiB | 52.13 MiB/s
2022-02-17T12:32:46.2942948Z Receiving objects:  79% (23958/30326), 130.32 MiB | 52.13 MiB/s
2022-02-17T12:32:46.3256738Z Receiving objects:  79% (24074/30326), 130.32 MiB | 52.13 MiB/s
2022-02-17T12:32:46.3721147Z Receiving objects:  80% (24261/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.4175888Z Receiving objects:  81% (24565/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.4705282Z Receiving objects:  82% (24868/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.5213857Z Receiving objects:  83% (25171/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.5684010Z Receiving objects:  84% (25474/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.6189469Z Receiving objects:  85% (25778/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.6644523Z Receiving objects:  86% (26081/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.6946128Z Receiving objects:  87% (26384/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.7664998Z Receiving objects:  88% (26687/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.7693785Z Receiving objects:  89% (26991/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.7822670Z Receiving objects:  90% (27294/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.7840611Z Receiving objects:  91% (27597/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.7870785Z Receiving objects:  92% (27900/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.7886652Z Receiving objects:  93% (28204/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.7909507Z Receiving objects:  94% (28507/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.7926768Z Receiving objects:  95% (28810/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.7944217Z Receiving objects:  96% (29113/30326), 161.27 MiB | 53.76 MiB/s
2022-02-17T12:32:46.7961156Z Receiving objects:  97% (29417/30326), 191.00 MiB | 54.57 MiB/s
2022-02-17T12:32:46.7977271Z Receiving objects:  98% (29720/30326), 191.00 MiB | 54.57 MiB/s
2022-02-17T12:32:46.8011577Z Receiving objects:  99% (30023/30326), 191.00 MiB | 54.57 MiB/s
2022-02-17T12:32:46.8012480Z remote: Total 30326 (delta 3719), reused 25101 (delta 3503), pack-reused 0        
2022-02-17T12:32:46.8045797Z Receiving objects: 100% (30326/30326), 191.00 MiB | 54.57 MiB/s
2022-02-17T12:32:46.8046095Z Receiving objects: 100% (30326/30326), 191.26 MiB | 54.49 MiB/s, done.
2022-02-17T12:32:46.8061311Z Resolving deltas:   0% (0/3719)
2022-02-17T12:32:46.8070912Z Resolving deltas:   1% (38/3719)
2022-02-17T12:32:46.8097691Z Resolving deltas:   2% (75/3719)
2022-02-17T12:32:46.8112022Z Resolving deltas:   3% (112/3719)
2022-02-17T12:32:46.8125509Z Resolving deltas:   4% (149/3719)
2022-02-17T12:32:46.8155240Z Resolving deltas:   5% (186/3719)
2022-02-17T12:32:46.8169889Z Resolving deltas:   6% (224/3719)
2022-02-17T12:32:46.8189381Z Resolving deltas:   7% (261/3719)
2022-02-17T12:32:46.8213596Z Resolving deltas:   8% (298/3719)
2022-02-17T12:32:46.8223009Z Resolving deltas:   9% (335/3719)
2022-02-17T12:32:46.8232586Z Resolving deltas:  10% (372/3719)
2022-02-17T12:32:46.8253512Z Resolving deltas:  11% (410/3719)
2022-02-17T12:32:46.8264369Z Resolving deltas:  12% (447/3719)
2022-02-17T12:32:46.8273425Z Resolving deltas:  13% (484/3719)
2022-02-17T12:32:46.8287959Z Resolving deltas:  14% (521/3719)
2022-02-17T12:32:46.8315895Z Resolving deltas:  15% (558/3719)
2022-02-17T12:32:46.8322855Z Resolving deltas:  16% (596/3719)
2022-02-17T12:32:46.8328119Z Resolving deltas:  17% (633/3719)
2022-02-17T12:32:46.8383241Z Resolving deltas:  18% (670/3719)
2022-02-17T12:32:46.8388075Z Resolving deltas:  19% (707/3719)
2022-02-17T12:32:46.8394838Z Resolving deltas:  20% (744/3719)
2022-02-17T12:32:46.8399091Z Resolving deltas:  21% (781/3719)
2022-02-17T12:32:46.8403161Z Resolving deltas:  22% (819/3719)
2022-02-17T12:32:46.8407304Z Resolving deltas:  23% (856/3719)
2022-02-17T12:32:46.8411345Z Resolving deltas:  24% (893/3719)
2022-02-17T12:32:46.8415153Z Resolving deltas:  25% (930/3719)
2022-02-17T12:32:46.8419298Z Resolving deltas:  26% (967/3719)
2022-02-17T12:32:46.8423178Z Resolving deltas:  27% (1005/3719)
2022-02-17T12:32:46.8426942Z Resolving deltas:  28% (1042/3719)
2022-02-17T12:32:46.8430634Z Resolving deltas:  29% (1079/3719)
2022-02-17T12:32:46.8434393Z Resolving deltas:  30% (1116/3719)
2022-02-17T12:32:46.8438551Z Resolving deltas:  31% (1153/3719)
2022-02-17T12:32:46.8443098Z Resolving deltas:  32% (1191/3719)
2022-02-17T12:32:46.8447023Z Resolving deltas:  33% (1228/3719)
2022-02-17T12:32:46.8451066Z Resolving deltas:  34% (1265/3719)
2022-02-17T12:32:46.8455336Z Resolving deltas:  35% (1302/3719)
2022-02-17T12:32:46.8459625Z Resolving deltas:  36% (1339/3719)
2022-02-17T12:32:46.8463928Z Resolving deltas:  37% (1377/3719)
2022-02-17T12:32:46.8468293Z Resolving deltas:  38% (1414/3719)
2022-02-17T12:32:46.8472898Z Resolving deltas:  39% (1451/3719)
2022-02-17T12:32:46.8476783Z Resolving deltas:  40% (1488/3719)
2022-02-17T12:32:46.8480874Z Resolving deltas:  41% (1525/3719)
2022-02-17T12:32:46.8484948Z Resolving deltas:  42% (1562/3719)
2022-02-17T12:32:46.8489062Z Resolving deltas:  43% (1600/3719)
2022-02-17T12:32:46.8492922Z Resolving deltas:  44% (1637/3719)
2022-02-17T12:32:46.8496961Z Resolving deltas:  45% (1674/3719)
2022-02-17T12:32:46.8500564Z Resolving deltas:  46% (1711/3719)
2022-02-17T12:32:46.8505344Z Resolving deltas:  47% (1748/3719)
2022-02-17T12:32:46.8509920Z Resolving deltas:  48% (1786/3719)
2022-02-17T12:32:46.8513807Z Resolving deltas:  49% (1823/3719)
2022-02-17T12:32:46.8518214Z Resolving deltas:  50% (1860/3719)
2022-02-17T12:32:46.8522674Z Resolving deltas:  51% (1897/3719)
2022-02-17T12:32:46.8526718Z Resolving deltas:  52% (1934/3719)
2022-02-17T12:32:46.8530577Z Resolving deltas:  53% (1972/3719)
2022-02-17T12:32:46.8534213Z Resolving deltas:  54% (2009/3719)
2022-02-17T12:32:46.8538314Z Resolving deltas:  55% (2046/3719)
2022-02-17T12:32:46.8545900Z Resolving deltas:  56% (2083/3719)
2022-02-17T12:32:46.8549610Z Resolving deltas:  57% (2120/3719)
2022-02-17T12:32:46.8552860Z Resolving deltas:  58% (2158/3719)
2022-02-17T12:32:46.8557909Z Resolving deltas:  59% (2195/3719)
2022-02-17T12:32:46.8561279Z Resolving deltas:  60% (2232/3719)
2022-02-17T12:32:46.8564767Z Resolving deltas:  61% (2269/3719)
2022-02-17T12:32:46.8568089Z Resolving deltas:  62% (2306/3719)
2022-02-17T12:32:46.8572488Z Resolving deltas:  63% (2343/3719)
2022-02-17T12:32:46.8576279Z Resolving deltas:  64% (2381/3719)
2022-02-17T12:32:46.8579870Z Resolving deltas:  65% (2418/3719)
2022-02-17T12:32:46.8583860Z Resolving deltas:  66% (2455/3719)
2022-02-17T12:32:46.8612062Z Resolving deltas:  67% (2492/3719)
2022-02-17T12:32:46.8616157Z Resolving deltas:  68% (2529/3719)
2022-02-17T12:32:46.8620106Z Resolving deltas:  69% (2567/3719)
2022-02-17T12:32:46.8624257Z Resolving deltas:  70% (2604/3719)
2022-02-17T12:32:46.8628630Z Resolving deltas:  71% (2641/3719)
2022-02-17T12:32:46.8637879Z Resolving deltas:  72% (2678/3719)
2022-02-17T12:32:46.8641088Z Resolving deltas:  73% (2715/3719)
2022-02-17T12:32:46.8643895Z Resolving deltas:  74% (2753/3719)
2022-02-17T12:32:46.8646491Z Resolving deltas:  75% (2790/3719)
2022-02-17T12:32:46.8649519Z Resolving deltas:  76% (2827/3719)
2022-02-17T12:32:46.8652386Z Resolving deltas:  77% (2864/3719)
2022-02-17T12:32:46.8655236Z Resolving deltas:  78% (2901/3719)
2022-02-17T12:32:46.8658649Z Resolving deltas:  79% (2939/3719)
2022-02-17T12:32:46.8661155Z Resolving deltas:  80% (2976/3719)
2022-02-17T12:32:46.8663648Z Resolving deltas:  81% (3013/3719)
2022-02-17T12:32:46.8666262Z Resolving deltas:  82% (3051/3719)
2022-02-17T12:32:46.8669331Z Resolving deltas:  83% (3087/3719)
2022-02-17T12:32:46.8672170Z Resolving deltas:  84% (3124/3719)
2022-02-17T12:32:46.8676246Z Resolving deltas:  85% (3162/3719)
2022-02-17T12:32:46.8681592Z Resolving deltas:  86% (3199/3719)
2022-02-17T12:32:46.8684000Z Resolving deltas:  87% (3236/3719)
2022-02-17T12:32:46.8687138Z Resolving deltas:  88% (3273/3719)
2022-02-17T12:32:46.8689685Z Resolving deltas:  89% (3310/3719)
2022-02-17T12:32:46.8693147Z Resolving deltas:  90% (3348/3719)
2022-02-17T12:32:46.8695913Z Resolving deltas:  91% (3385/3719)
2022-02-17T12:32:46.8698318Z Resolving deltas:  92% (3422/3719)
2022-02-17T12:32:46.8700826Z Resolving deltas:  93% (3459/3719)
2022-02-17T12:32:46.8704212Z Resolving deltas:  94% (3496/3719)
2022-02-17T12:32:46.8706479Z Resolving deltas:  95% (3534/3719)
2022-02-17T12:32:46.8709783Z Resolving deltas:  96% (3571/3719)
2022-02-17T12:32:46.8712358Z Resolving deltas:  97% (3608/3719)
2022-02-17T12:32:46.8715413Z Resolving deltas:  98% (3645/3719)
2022-02-17T12:32:46.8768815Z Resolving deltas:  99% (3682/3719)
2022-02-17T12:32:46.8769253Z Resolving deltas: 100% (3719/3719)
2022-02-17T12:32:46.8769831Z Resolving deltas: 100% (3719/3719), done.
2022-02-17T12:32:47.5960472Z From https://github.com/alexanderramos89/asset
2022-02-17T12:32:47.5961485Z  * [new branch]        alexanderramos89-patch-5 -> origin/alexanderramos89-patch-5
2022-02-17T12:32:47.5981890Z ##[endgroup]
2022-02-17T12:32:47.5982429Z ##[group]Determining the checkout info
2022-02-17T12:32:47.5984030Z ##[endgroup]
2022-02-17T12:32:47.5984458Z ##[group]Checking out the ref
2022-02-17T12:32:47.5988766Z [command]/usr/bin/git checkout --progress --force -B alexanderramos89-patch-5 refs/remotes/origin/alexanderramos89-patch-5
2022-02-17T12:32:48.6505711Z Updating files:  56% (12302/21809)
2022-02-17T12:32:48.6671829Z Updating files:  57% (12432/21809)
2022-02-17T12:32:48.6830658Z Updating files:  58% (12650/21809)
2022-02-17T12:32:48.7059708Z Updating files:  59% (12868/21809)
2022-02-17T12:32:48.7265233Z Updating files:  60% (13086/21809)
2022-02-17T12:32:48.7420634Z Updating files:  61% (13304/21809)
2022-02-17T12:32:48.7609700Z Updating files:  62% (13522/21809)
2022-02-17T12:32:48.7867238Z Updating files:  63% (13740/21809)
2022-02-17T12:32:48.8090834Z Updating files:  64% (13958/21809)
2022-02-17T12:32:48.8292173Z Updating files:  65% (14176/21809)
2022-02-17T12:32:48.8507292Z Updating files:  66% (14394/21809)
2022-02-17T12:32:48.8716449Z Updating files:  67% (14613/21809)
2022-02-17T12:32:48.8931083Z Updating files:  68% (14831/21809)
2022-02-17T12:32:48.9139934Z Updating files:  69% (15049/21809)
2022-02-17T12:32:48.9355071Z Updating files:  70% (15267/21809)
2022-02-17T12:32:48.9604754Z Updating files:  71% (15485/21809)
2022-02-17T12:32:48.9831721Z Updating files:  72% (15703/21809)
2022-02-17T12:32:49.0023977Z Updating files:  73% (15921/21809)
2022-02-17T12:32:49.0242238Z Updating files:  74% (16139/21809)
2022-02-17T12:32:49.0432881Z Updating files:  75% (16357/21809)
2022-02-17T12:32:49.0641929Z Updating files:  76% (16575/21809)
2022-02-17T12:32:49.0877402Z Updating files:  77% (16793/21809)
2022-02-17T12:32:49.1104704Z Updating files:  78% (17012/21809)
2022-02-17T12:32:49.1359316Z Updating files:  79% (17230/21809)
2022-02-17T12:32:49.1558279Z Updating files:  80% (17448/21809)
2022-02-17T12:32:49.1741366Z Updating files:  81% (17666/21809)
2022-02-17T12:32:49.1950073Z Updating files:  82% (17884/21809)
2022-02-17T12:32:49.2133779Z Updating files:  83% (18102/21809)
2022-02-17T12:32:49.2199024Z Updating files:  84% (18320/21809)
2022-02-17T12:32:49.2257506Z Updating files:  85% (18538/21809)
2022-02-17T12:32:49.2362373Z Updating files:  86% (18756/21809)
2022-02-17T12:32:49.2428512Z Updating files:  87% (18974/21809)
2022-02-17T12:32:49.2490452Z Updating files:  88% (19192/21809)
2022-02-17T12:32:49.2579147Z Updating files:  89% (19411/21809)
2022-02-17T12:32:49.2639978Z Updating files:  90% (19629/21809)
2022-02-17T12:32:49.2706105Z Updating files:  91% (19847/21809)
2022-02-17T12:32:49.2774443Z Updating files:  92% (20065/21809)
2022-02-17T12:32:49.2835925Z Updating files:  93% (20283/21809)
2022-02-17T12:32:49.2898349Z Updating files:  94% (20501/21809)
2022-02-17T12:32:49.2960310Z Updating files:  95% (20719/21809)
2022-02-17T12:32:49.3022684Z Updating files:  96% (20937/21809)
2022-02-17T12:32:49.3083830Z Updating files:  97% (21155/21809)
2022-02-17T12:32:49.3146985Z Updating files:  98% (21373/21809)
2022-02-17T12:32:49.3231583Z Updating files:  99% (21591/21809)
2022-02-17T12:32:49.3232013Z Updating files: 100% (21809/21809)
2022-02-17T12:32:49.3232585Z Updating files: 100% (21809/21809), done.
2022-02-17T12:32:49.3704622Z Switched to a new branch 'alexanderramos89-patch-5'
2022-02-17T12:32:49.3708651Z branch 'alexanderramos89-patch-5' set up to track 'origin/alexanderramos89-patch-5'.
2022-02-17T12:32:49.3811850Z ##[endgroup]
2022-02-17T12:32:49.3852968Z [command]/usr/bin/git log -1 --format='%H'
2022-02-17T12:32:49.3879414Z '81a6507e714be0a789adfc564fb80844911ad974'
2022-02-17T12:32:49.4020821Z ##[group]Run actions/setup-node@v1
2022-02-17T12:32:49.4021024Z with:
2022-02-17T12:32:49.4021198Z   node-version: 12
2022-02-17T12:32:49.4021402Z   always-auth: false
2022-02-17T12:32:49.4021576Z ##[endgroup]
2022-02-17T12:32:49.5177965Z [command]/opt/hostedtoolcache/node/12.22.10/x64/bin/node --version
2022-02-17T12:32:49.5536629Z v12.22.10
2022-02-17T12:32:49.5559523Z [command]/opt/hostedtoolcache/node/12.22.10/x64/bin/npm --version
2022-02-17T12:32:49.7675061Z 6.14.16
2022-02-17T12:32:49.7890579Z ##[group]Run npm ci
2022-02-17T12:32:49.7890834Z [36;1mnpm ci[0m
2022-02-17T12:32:49.7943765Z shell: /usr/bin/bash -e {0}
2022-02-17T12:32:49.7944004Z ##[endgroup]
2022-02-17T12:32:56.8397239Z 
2022-02-17T12:32:56.8404416Z > protobufjs@6.11.2 postinstall /home/runner/work/asset/asset/node_modules/protobufjs
2022-02-17T12:32:56.8405128Z > node scripts/postinstall
2022-02-17T12:32:56.8405347Z 
2022-02-17T12:32:56.9959982Z 
2022-02-17T12:32:56.9960501Z > sharp@0.28.3 install /home/runner/work/asset/asset/node_modules/sharp
2022-02-17T12:32:56.9961450Z > (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)
2022-02-17T12:32:56.9961753Z 
2022-02-17T12:33:08.7713986Z sharp: Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.10.6/libvips-8.10.6-linux-x64.tar.br
2022-02-17T12:33:10.2244164Z 
2022-02-17T12:33:10.2245032Z > keccak@1.4.0 install /home/runner/work/asset/asset/node_modules/keccak
2022-02-17T12:33:10.2246109Z > npm run rebuild || echo "Keccak bindings compilation fail. Pure JS implementation will be used."
2022-02-17T12:33:10.2246497Z 
2022-02-17T12:33:10.4171656Z 
2022-02-17T12:33:10.4181017Z > keccak@1.4.0 rebuild /home/runner/work/asset/asset/node_modules/keccak
2022-02-17T12:33:10.4185834Z > node-gyp rebuild
2022-02-17T12:33:10.4186181Z 
2022-02-17T12:33:13.0940258Z make: Entering directory '/home/runner/work/asset/asset/node_modules/keccak/build'
2022-02-17T12:33:13.1093839Z   CXX(target) Release/obj.target/keccak/src/addon.o
2022-02-17T12:33:15.9507821Z In file included from ../src/addon.cc:2:
2022-02-17T12:33:15.9509319Z ../../nan/nan.h: In function ‘void Nan::AsyncQueueWorker(Nan::AsyncWorker*)’:
2022-02-17T12:33:15.9601534Z ../../nan/nan.h:2298:62: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
2022-02-17T12:33:15.9602119Z  2298 |     , reinterpret_cast<uv_after_work_cb>(AsyncExecuteComplete)
2022-02-17T12:33:15.9602705Z       |                                                              ^
2022-02-17T12:33:15.9693146Z ../src/addon.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE KeccakWrapper::Initialize(Nan::NAN_METHOD_ARGS_TYPE)’:
2022-02-17T12:33:15.9693884Z ../src/addon.cc:37:47: error: no matching function for call to ‘v8::Value::IntegerValue()’
2022-02-17T12:33:15.9694332Z    37 |     unsigned int rate = info[0]->IntegerValue();
2022-02-17T12:33:15.9694626Z       |                                               ^
2022-02-17T12:33:15.9695099Z In file included from /home/runner/.cache/node-gyp/12.22.10/include/node/node.h:67,
2022-02-17T12:33:15.9695452Z                  from ../src/addon.cc:1:
2022-02-17T12:33:15.9696217Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2705:40: note: candidate: ‘v8::Maybe<long int> v8::Value::IntegerValue(v8::Local<v8::Context>) const’
2022-02-17T12:33:15.9696682Z  2705 |   V8_WARN_UNUSED_RESULT Maybe<int64_t> IntegerValue(
2022-02-17T12:33:15.9696980Z       |                                        ^~~~~~~~~~~~
2022-02-17T12:33:15.9697544Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2705:40: note:   candidate expects 1 argument, 0 provided
2022-02-17T12:33:15.9698175Z ../src/addon.cc:38:51: error: no matching function for call to ‘v8::Value::IntegerValue()’
2022-02-17T12:33:15.9698620Z    38 |     unsigned int capacity = info[1]->IntegerValue();
2022-02-17T12:33:15.9698902Z       |                                                   ^
2022-02-17T12:33:15.9699380Z In file included from /home/runner/.cache/node-gyp/12.22.10/include/node/node.h:67,
2022-02-17T12:33:15.9699708Z                  from ../src/addon.cc:1:
2022-02-17T12:33:15.9700426Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2705:40: note: candidate: ‘v8::Maybe<long int> v8::Value::IntegerValue(v8::Local<v8::Context>) const’
2022-02-17T12:33:15.9701153Z  2705 |   V8_WARN_UNUSED_RESULT Maybe<int64_t> IntegerValue(
2022-02-17T12:33:15.9701461Z       |                                        ^~~~~~~~~~~~
2022-02-17T12:33:15.9702018Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2705:40: note:   candidate expects 1 argument, 0 provided
2022-02-17T12:33:15.9702807Z ../src/addon.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE KeccakWrapper::AbsorbLastFewBits(Nan::NAN_METHOD_ARGS_TYPE)’:
2022-02-17T12:33:15.9703491Z ../src/addon.cc:56:48: error: no matching function for call to ‘v8::Value::IntegerValue()’
2022-02-17T12:33:15.9703922Z    56 |     unsigned char bits = info[0]->IntegerValue();
2022-02-17T12:33:15.9704211Z       |                                                ^
2022-02-17T12:33:15.9704696Z In file included from /home/runner/.cache/node-gyp/12.22.10/include/node/node.h:67,
2022-02-17T12:33:15.9705008Z                  from ../src/addon.cc:1:
2022-02-17T12:33:15.9705756Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2705:40: note: candidate: ‘v8::Maybe<long int> v8::Value::IntegerValue(v8::Local<v8::Context>) const’
2022-02-17T12:33:15.9706190Z  2705 |   V8_WARN_UNUSED_RESULT Maybe<int64_t> IntegerValue(
2022-02-17T12:33:15.9706473Z       |                                        ^~~~~~~~~~~~
2022-02-17T12:33:15.9707024Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2705:40: note:   candidate expects 1 argument, 0 provided
2022-02-17T12:33:15.9707754Z ../src/addon.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE KeccakWrapper::Squeeze(Nan::NAN_METHOD_ARGS_TYPE)’:
2022-02-17T12:33:15.9708409Z ../src/addon.cc:64:43: error: no matching function for call to ‘v8::Value::IntegerValue()’
2022-02-17T12:33:15.9708929Z    64 |     size_t length = info[0]->IntegerValue();
2022-02-17T12:33:15.9709195Z       |                                           ^
2022-02-17T12:33:15.9709690Z In file included from /home/runner/.cache/node-gyp/12.22.10/include/node/node.h:67,
2022-02-17T12:33:15.9710009Z                  from ../src/addon.cc:1:
2022-02-17T12:33:15.9710728Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2705:40: note: candidate: ‘v8::Maybe<long int> v8::Value::IntegerValue(v8::Local<v8::Context>) const’
2022-02-17T12:33:15.9711158Z  2705 |   V8_WARN_UNUSED_RESULT Maybe<int64_t> IntegerValue(
2022-02-17T12:33:15.9711461Z       |                                        ^~~~~~~~~~~~
2022-02-17T12:33:15.9712004Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2705:40: note:   candidate expects 1 argument, 0 provided
2022-02-17T12:33:15.9712729Z ../src/addon.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE KeccakWrapper::Copy(Nan::NAN_METHOD_ARGS_TYPE)’:
2022-02-17T12:33:15.9713427Z ../src/addon.cc:75:82: error: no matching function for call to ‘v8::Value::ToObject()’
2022-02-17T12:33:15.9713947Z    75 |     KeccakWrapper* to = Nan::ObjectWrap::Unwrap<KeccakWrapper>(info[0]->ToObject());
2022-02-17T12:33:15.9714314Z       |                                                                                  ^
2022-02-17T12:33:15.9714802Z In file included from /home/runner/.cache/node-gyp/12.22.10/include/node/node.h:67,
2022-02-17T12:33:15.9715110Z                  from ../src/addon.cc:1:
2022-02-17T12:33:15.9715847Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2672:44: note: candidate: ‘v8::MaybeLocal<v8::Object> v8::Value::ToObject(v8::Local<v8::Context>) const’
2022-02-17T12:33:15.9716283Z  2672 |   V8_WARN_UNUSED_RESULT MaybeLocal<Object> ToObject(
2022-02-17T12:33:15.9716563Z       |                                            ^~~~~~~~
2022-02-17T12:33:15.9717105Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2672:44: note:   candidate expects 1 argument, 0 provided
2022-02-17T12:33:15.9717685Z In file included from /home/runner/.cache/node-gyp/12.22.10/include/node/v8-internal.h:14,
2022-02-17T12:33:15.9718169Z                  from /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:27,
2022-02-17T12:33:15.9718685Z                  from /home/runner/.cache/node-gyp/12.22.10/include/node/node.h:67,
2022-02-17T12:33:15.9718979Z                  from ../src/addon.cc:1:
2022-02-17T12:33:15.9719672Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2686:31: note: candidate: ‘v8::Local<v8::Object> v8::Value::ToObject(v8::Isolate*) const’
2022-02-17T12:33:15.9720097Z  2686 |                 Local<Object> ToObject(Isolate* isolate) const);
2022-02-17T12:33:15.9720372Z       |                               ^~~~~~~~
2022-02-17T12:33:15.9720939Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8config.h:328:3: note: in definition of macro ‘V8_DEPRECATED’
2022-02-17T12:33:15.9721335Z   328 |   declarator __attribute__((deprecated(message)))
2022-02-17T12:33:15.9721567Z       |   ^~~~~~~~~~
2022-02-17T12:33:15.9722090Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8.h:2686:31: note:   candidate expects 1 argument, 0 provided
2022-02-17T12:33:15.9722458Z  2686 |                 Local<Object> ToObject(Isolate* isolate) const);
2022-02-17T12:33:15.9722749Z       |                               ^~~~~~~~
2022-02-17T12:33:15.9723313Z /home/runner/.cache/node-gyp/12.22.10/include/node/v8config.h:328:3: note: in definition of macro ‘V8_DEPRECATED’
2022-02-17T12:33:15.9723696Z   328 |   declarator __attribute__((deprecated(message)))
2022-02-17T12:33:15.9723926Z       |   ^~~~~~~~~~
2022-02-17T12:33:15.9724210Z In file included from ../src/addon.cc:1:
2022-02-17T12:33:15.9724487Z ../src/addon.cc: At global scope:
2022-02-17T12:33:15.9725895Z /home/runner/.cache/node-gyp/12.22.10/include/node/node.h:736:43: warning: cast between incompatible function types from ‘void (*)(Nan::ADDON_REGISTER_FUNCTION_ARGS_TYPE, Nan::ADDON_REGISTER_FUNCTION_ARGS_TYPE)’ {aka ‘void (*)(v8::Local<v8::Object>, v8::Local<v8::Object>)’} to ‘node::addon_register_func’ {aka ‘void (*)(v8::Local<v8::Object>, v8::Local<v8::Value>, void*)’} [-Wcast-function-type]
2022-02-17T12:33:15.9726739Z   736 |       (node::addon_register_func) (regfunc),                          \
2022-02-17T12:33:15.9727031Z       |                                           ^
2022-02-17T12:33:15.9727570Z /home/runner/.cache/node-gyp/12.22.10/include/node/node.h:770:3: note: in expansion of macro ‘NODE_MODULE_X’
2022-02-17T12:33:15.9728011Z   770 |   NODE_MODULE_X(modname, regfunc, NULL, 0)  // NOLINT (readability/null_usage)
2022-02-17T12:33:15.9728381Z       |   ^~~~~~~~~~~~~
2022-02-17T12:33:15.9728793Z ../src/addon.cc:86:1: note: in expansion of macro ‘NODE_MODULE’
2022-02-17T12:33:15.9729082Z    86 | NODE_MODULE(keccak, Init)
2022-02-17T12:33:15.9729275Z       | ^~~~~~~~~~~
2022-02-17T12:33:16.1463652Z make: *** [keccak.target.mk:129: Release/obj.target/keccak/src/addon.o] Error 1
2022-02-17T12:33:16.1470931Z make: Leaving directory '/home/runner/work/asset/asset/node_modules/keccak/build'
2022-02-17T12:33:16.1471538Z gyp ERR! build error 
2022-02-17T12:33:16.1475735Z gyp ERR! stack Error: `make` failed with exit code: 2
2022-02-17T12:33:16.1478232Z gyp ERR! stack     at ChildProcess.onExit (/opt/hostedtoolcache/node/12.22.10/x64/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
2022-02-17T12:33:16.1478906Z gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
2022-02-17T12:33:16.1479497Z gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
2022-02-17T12:33:16.1520269Z gyp ERR! System Linux 5.11.0-1028-azure
2022-02-17T12:33:16.1524651Z gyp ERR! command "/opt/hostedtoolcache/node/12.22.10/x64/bin/node" "/opt/hostedtoolcache/node/12.22.10/x64/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
2022-02-17T12:33:16.1525158Z gyp ERR! cwd /home/runner/work/asset/asset/node_modules/keccak
2022-02-17T12:33:16.1525494Z gyp ERR! node -v v12.22.10
2022-02-17T12:33:16.1525799Z gyp ERR! node-gyp -v v5.1.0
2022-02-17T12:33:16.1526018Z gyp ERR! not ok 
2022-02-17T12:33:16.1549866Z npm ERR! code ELIFECYCLE
2022-02-17T12:33:16.1550616Z npm ERR! errno 1
2022-02-17T12:33:16.1609943Z npm ERR! keccak@1.4.0 rebuild: `node-gyp rebuild`
2022-02-17T12:33:16.1612750Z npm ERR! Exit status 1
2022-02-17T12:33:16.1615724Z npm ERR! 
2022-02-17T12:33:16.1618384Z npm ERR! Failed at the keccak@1.4.0 rebuild script.
2022-02-17T12:33:16.1621120Z npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2022-02-17T12:33:16.1675190Z npm WARN Local package.json exists, but node_modules missing, did you mean to install?
2022-02-17T12:33:16.1680672Z 
2022-02-17T12:33:16.1683650Z npm ERR! A complete log of this run can be found in:
2022-02-17T12:33:16.1686495Z npm ERR!     /home/runner/.npm/_cacache/_logs/2022-02-17T12_33_16_162Z-debug.log
2022-02-17T12:33:16.1729460Z Keccak bindings compilation fail. Pure JS implementation will be used.
2022-02-17T12:33:16.2329345Z added 989 packages in 26.024s
2022-02-17T12:33:16.2458844Z ##[group]Run npm run fix-sanity
2022-02-17T12:33:16.2459082Z [36;1mnpm run fix-sanity[0m
2022-02-17T12:33:16.2506243Z shell: /usr/bin/bash -e {0}
2022-02-17T12:33:16.2506460Z ##[endgroup]
2022-02-17T12:33:16.4411768Z 
2022-02-17T12:33:16.4412553Z > assets@1.0.0 fix-sanity /home/runner/work/asset/asset
2022-02-17T12:33:16.4413161Z > ts-node ./script/entrypoint/fix-sanity
2022-02-17T12:33:16.4413545Z 
2022-02-17T12:33:19.9836765Z Running sanity fixes...
2022-02-17T12:33:19.9953656Z Sanity fix 'Ethereum forks':
2022-02-17T12:33:19.9957277Z Checking for checksum formats ...
2022-02-17T12:33:20.6545460Z Sanity fix 'Logo sizes':
2022-02-17T12:33:20.6556743Z Checking all logos for size ...
2022-02-17T12:33:21.0866889Z Checking logos completed, 9396 logos checked, 0 too large, 0 logos updated
2022-02-17T12:33:21.0871850Z Sanity fix 'Validators':
2022-02-17T12:33:21.0881624Z Formatted json file /home/runner/work/asset/asset/blockchains/tezos/validators/list.json
2022-02-17T12:33:21.0886015Z Formatted json file /home/runner/work/asset/asset/blockchains/cosmos/validators/list.json
2022-02-17T12:33:21.0889341Z Formatted json file /home/runner/work/asset/asset/blockchains/iotex/validators/list.json
2022-02-17T12:33:21.0890883Z Formatted json file /home/runner/work/asset/asset/blockchains/tron/validators/list.json
2022-02-17T12:33:21.0892773Z Formatted json file /home/runner/work/asset/asset/blockchains/waves/validators/list.json
2022-02-17T12:33:21.0895158Z Formatted json file /home/runner/work/asset/asset/blockchains/kava/validators/list.json
2022-02-17T12:33:21.0896208Z Formatted json file /home/runner/work/asset/asset/blockchains/terra/validators/list.json
2022-02-17T12:33:21.0898072Z Formatted json file /home/runner/work/asset/asset/blockchains/binance/validators/list.json
2022-02-17T12:33:21.0899783Z Sanity fix 'Json files':
2022-02-17T12:33:21.0903935Z Formatting json files...
2022-02-17T12:33:21.6592114Z Formatted 0 json files (total 8842)
2022-02-17T12:33:21.6609364Z All sanity fixes done.
2022-02-17T12:33:21.6921286Z ##[group]Run echo "GITHUB_REF " $GITHUB_REF "  GITHUB_BASE_REF " $GITHUB_BASE_REF
2022-02-17T12:33:21.6921630Z [36;1mecho "GITHUB_REF " $GITHUB_REF "  GITHUB_BASE_REF " $GITHUB_BASE_REF[0m
2022-02-17T12:33:21.6968716Z shell: /usr/bin/bash -e {0}
2022-02-17T12:33:21.6968926Z ##[endgroup]
2022-02-17T12:33:21.7036639Z GITHUB_REF  refs/heads/alexanderramos89-patch-5   GITHUB_BASE_REF 
2022-02-17T12:33:21.7064888Z ##[group]Run git status
2022-02-17T12:33:21.7065100Z [36;1mgit status[0m
2022-02-17T12:33:21.7065276Z [36;1mgit diff[0m
2022-02-17T12:33:21.7110395Z shell: /usr/bin/bash -e {0}
2022-02-17T12:33:21.7110604Z ##[endgroup]
2022-02-17T12:33:22.0532778Z On branch alexanderramos89-patch-5
2022-02-17T12:33:22.0538527Z Your branch is up to date with 'origin/alexanderramos89-patch-5'.
2022-02-17T12:33:22.0539167Z 
2022-02-17T12:33:22.0539523Z nothing to commit, working tree clean
2022-02-17T12:33:22.0844984Z ##[group]Run npm run check
2022-02-17T12:33:22.0845253Z [36;1mnpm run check[0m
2022-02-17T12:33:22.0892857Z shell: /usr/bin/bash -e {0}
2022-02-17T12:33:22.0893232Z ##[endgroup]
2022-02-17T12:33:22.2802263Z 
2022-02-17T12:33:22.2803126Z > assets@1.0.0 check /home/runner/work/asset/asset
2022-02-17T12:33:22.2803900Z > ts-node ./script/entrypoint/check
2022-02-17T12:33:22.2804098Z 
2022-02-17T12:33:26.1317731Z Running sanity checks...
2022-02-17T12:33:26.1483294Z    Action 'Folders and Files' has 6 check steps
2022-02-17T12:33:26.1494352Z -  XXX 'Repository root dir':  1 errors
2022-02-17T12:33:26.1495184Z    XXX   'File "SECURITY.md" should not be in root or added to predifined list'
2022-02-17T12:33:26.1512047Z -  ✓ 'Chain folders are lowercase, contain only predefined list of files' OK
2022-02-17T12:33:26.1626689Z -  ✓ 'Chain folders have logo, and correct size' OK
2022-02-17T12:33:26.2286710Z -  !! 'Asset folders contain logo and info':  5 warnings
2022-02-17T12:33:26.2287917Z    !!   'Missing info file for asset 'terra/ukrw' -- /home/runner/work/asset/asset/blockchains/terra/assets/ukrw/info.json'
2022-02-17T12:33:26.2289034Z    !!   'Missing info file for asset 'terra/uluna' -- /home/runner/work/asset/asset/blockchains/terra/assets/uluna/info.json'
2022-02-17T12:33:26.2289994Z    !!   'Missing info file for asset 'terra/umnt' -- /home/runner/work/asset/asset/blockchains/terra/assets/umnt/info.json'
2022-02-17T12:33:26.2290713Z    !!   'Missing info file for asset 'terra/usdr' -- /home/runner/work/asset/asset/blockchains/terra/assets/usdr/info.json'
2022-02-17T12:33:26.2291580Z    !!   'Missing info file for asset 'terra/uusd' -- /home/runner/work/asset/asset/blockchains/terra/assets/uusd/info.json'
2022-02-17T12:33:26.3225213Z -  ✓ 'Asset folders contain only predefined set of files' OK
2022-02-17T12:33:26.3231365Z -  ✓ 'Dapps folders contain only .png files, with all lowercase names' OK
2022-02-17T12:33:26.3237295Z    Action 'Asset Infos' has 23 check steps
2022-02-17T12:33:26.3463403Z -  ✓ 'Token info.json's for chain binance' OK
2022-02-17T12:33:26.3587685Z -  ✓ 'Token info.json's for chain classic' OK
2022-02-17T12:33:26.3593016Z -  ✓ 'Token info.json's for chain eos' OK
2022-02-17T12:33:26.7461121Z -  ✓ 'Token info.json's for chain ethereum' OK
2022-02-17T12:33:26.7530082Z -  ✓ 'Token info.json's for chain gochain' OK
2022-02-17T12:33:26.7533019Z -  ✓ 'Token info.json's for chain kava' OK
2022-02-17T12:33:26.7535506Z -  ✓ 'Token info.json's for chain neo' OK
2022-02-17T12:33:26.7537153Z -  ✓ 'Token info.json's for chain nuls' OK
2022-02-17T12:33:26.7538860Z -  ✓ 'Token info.json's for chain ontology' OK
2022-02-17T12:33:26.7541978Z -  ✓ 'Token info.json's for chain poa' OK
2022-02-17T12:33:26.7585521Z -  ✓ 'Token info.json's for chain polygon' OK
2022-02-17T12:33:26.8301219Z -  ✓ 'Token info.json's for chain smartchain' OK
2022-02-17T12:33:26.8314047Z -  ✓ 'Token info.json's for chain solana' OK
2022-02-17T12:33:26.8315891Z -  ✓ 'Token info.json's for chain terra' OK
2022-02-17T12:33:26.8317450Z -  ✓ 'Token info.json's for chain theta' OK
2022-02-17T12:33:26.8326214Z -  ✓ 'Token info.json's for chain thundertoken' OK
2022-02-17T12:33:26.8330331Z -  ✓ 'Token info.json's for chain tomochain' OK
2022-02-17T12:33:26.8417023Z -  ✓ 'Token info.json's for chain tron' OK
2022-02-17T12:33:26.8418932Z -  ✓ 'Token info.json's for chain vechain' OK
2022-02-17T12:33:26.8422790Z -  ✓ 'Token info.json's for chain wanchain' OK
2022-02-17T12:33:26.8438764Z -  ✓ 'Token info.json's for chain waves' OK
2022-02-17T12:33:26.8443346Z -  ✓ 'Token info.json's for chain xdai' OK
2022-02-17T12:33:26.8484314Z -  ✓ 'Coin info.json's' OK
2022-02-17T12:33:26.8485099Z - ✓ Action 'Asset Infos' OK, all 23 steps
2022-02-17T12:33:26.8488752Z    Action 'Ethereum forks' has 9 check steps
2022-02-17T12:33:26.8553168Z      Found 6527 assets for chain ethereum
2022-02-17T12:33:27.0674299Z -  ✓ 'Folder structure for chain ethereum (ethereum fork)' OK
2022-02-17T12:33:27.0679820Z      Found 84 assets for chain classic
2022-02-17T12:33:27.0704417Z -  ✓ 'Folder structure for chain classic (ethereum fork)' OK
2022-02-17T12:33:27.0705255Z      Found 3 assets for chain poa
2022-02-17T12:33:27.0706455Z -  ✓ 'Folder structure for chain poa (ethereum fork)' OK
2022-02-17T12:33:27.0706981Z      Found 6 assets for chain tomochain
2022-02-17T12:33:27.0709300Z -  ✓ 'Folder structure for chain tomochain (ethereum fork)' OK
2022-02-17T12:33:27.0709902Z      Found 6 assets for chain gochain
2022-02-17T12:33:27.0712214Z -  ✓ 'Folder structure for chain gochain (ethereum fork)' OK
2022-02-17T12:33:27.0712752Z      Found 6 assets for chain wanchain
2022-02-17T12:33:27.0719267Z -  ✓ 'Folder structure for chain wanchain (ethereum fork)' OK
2022-02-17T12:33:27.0719778Z      Found 16 assets for chain thundertoken
2022-02-17T12:33:27.0724705Z -  ✓ 'Folder structure for chain thundertoken (ethereum fork)' OK
2022-02-17T12:33:27.0738590Z      Found 1557 assets for chain smartchain
2022-02-17T12:33:27.1110266Z -  ✓ 'Folder structure for chain smartchain (ethereum fork)' OK
2022-02-17T12:33:27.1111035Z      Found 38 assets for chain polygon
2022-02-17T12:33:27.1119986Z -  ✓ 'Folder structure for chain polygon (ethereum fork)' OK
2022-02-17T12:33:27.1120690Z - ✓ Action 'Ethereum forks' OK, all 9 steps
2022-02-17T12:33:27.1122050Z    Action 'Logo sizes' has 1 check steps
2022-02-17T12:33:27.1127143Z Checking all logos for size ...
2022-02-17T12:33:27.5263282Z Checking logos completed, 9396 logos checked, 0 too large, 0 logos updated
2022-02-17T12:33:27.5264586Z -  ✓ 'Check that logos are not too large' OK
2022-02-17T12:33:27.5265483Z - ✓ Action 'Logo sizes' OK, all 1 steps
2022-02-17T12:33:27.5266075Z    Action 'Validators' has 9 check steps
2022-02-17T12:33:27.5269457Z -  ✓ 'Make sure tests added for new staking chain' OK
2022-02-17T12:33:27.5300533Z -  ✓ 'Make sure chain tezos has valid list file, has logo' OK
2022-02-17T12:33:27.5332687Z -  ✓ 'Make sure chain cosmos has valid list file, has logo' OK
2022-02-17T12:33:27.5370877Z -  ✓ 'Make sure chain iotex has valid list file, has logo' OK
2022-02-17T12:33:27.5375514Z -  ✓ 'Make sure chain tron has valid list file, has logo' OK
2022-02-17T12:33:27.5384959Z -  ✓ 'Make sure chain waves has valid list file, has logo' OK
2022-02-17T12:33:27.5396864Z -  ✓ 'Make sure chain kava has valid list file, has logo' OK
2022-02-17T12:33:27.5403532Z -  ✓ 'Make sure chain terra has valid list file, has logo' OK
2022-02-17T12:33:27.5421545Z -  ✓ 'Make sure chain binance has valid list file, has logo' OK
2022-02-17T12:33:27.5422263Z - ✓ Action 'Validators' OK, all 9 steps
2022-02-17T12:33:27.5423390Z    Action 'Json files' has 1 check steps
2022-02-17T12:33:28.0375869Z -  ✓ 'Check all JSON files to have valid content' OK
2022-02-17T12:33:28.0391230Z - ✓ Action 'Json files' OK, all 1 steps
2022-02-17T12:33:28.0392587Z    Action 'Binance chain' has 1 check steps
2022-02-17T12:33:28.0397938Z Retrieving token infos
2022-02-17T12:33:28.6371325Z Using 533 assets
2022-02-17T12:33:28.6373040Z      197 assets checked.
2022-02-17T12:33:28.6373904Z -  ✓ 'Binance chain; assets must exist on chain' OK
2022-02-17T12:33:28.6375659Z - ✓ Action 'Binance chain' OK, all 1 steps
2022-02-17T12:33:28.6382129Z    Action 'Cosmos chain' has 1 check steps
2022-02-17T12:33:28.6388277Z -  ✓ 'Cosmos validator assets must have correct format' OK
2022-02-17T12:33:28.6388927Z - ✓ Action 'Cosmos chain' OK, all 1 steps
2022-02-17T12:33:28.6390056Z    Action 'Kava chain' has 1 check steps
2022-02-17T12:33:28.6394998Z -  ✓ 'Kava validator assets must have correct format' OK
2022-02-17T12:33:28.6395583Z - ✓ Action 'Kava chain' OK, all 1 steps
2022-02-17T12:33:28.6396596Z    Action 'Terra chain' has 1 check steps
2022-02-17T12:33:28.6401130Z -  ✓ 'Terra validator assets must have correct format' OK
2022-02-17T12:33:28.6401719Z - ✓ Action 'Terra chain' OK, all 1 steps
2022-02-17T12:33:28.6402470Z    Action 'Tezos' has 1 check steps
2022-02-17T12:33:28.6489835Z -  ✓ 'Tezos validator assets must have correct format' OK
2022-02-17T12:33:28.6490873Z - ✓ Action 'Tezos' OK, all 1 steps
2022-02-17T12:33:28.6491796Z    Action 'Tron chain' has 2 check steps
2022-02-17T12:33:28.6492451Z -  ✓ 'Tron assets should be TRC10 or TRC20, logo of correct size' OK
2022-02-17T12:33:28.6493310Z -  ✓ 'Tron validator assets must have correct format' OK
2022-02-17T12:33:28.6493941Z - ✓ Action 'Tron chain' OK, all 2 steps
2022-02-17T12:33:28.6494776Z    Action 'Waves chain' has 1 check steps
2022-02-17T12:33:28.6495376Z -  ✓ 'Waves validator assets must have correct format' OK
2022-02-17T12:33:28.6496172Z - ✓ Action 'Waves chain' OK, all 1 steps
2022-02-17T12:33:28.6496736Z All sanity checks done, found 1 errors, 5 warnings
2022-02-17T12:33:28.6497384Z Running consistency checks...
2022-02-17T12:33:28.6498033Z    Action 'Allowlists' has 19 check steps
2022-02-17T12:33:28.8775320Z -  ✓ 'Allowlist and denylist for ethereum should be consistent with assets' OK
2022-02-17T12:33:28.8789478Z -  ✓ 'Allowlist and denylist for classic should be consistent with assets' OK
2022-02-17T12:33:28.8794495Z -  ✓ 'Allowlist and denylist for poa should be consistent with assets' OK
2022-02-17T12:33:28.8798865Z -  ✓ 'Allowlist and denylist for tomochain should be consistent with assets' OK
2022-02-17T12:33:28.8803344Z -  ✓ 'Allowlist and denylist for gochain should be consistent with assets' OK
2022-02-17T12:33:28.8807988Z -  ✓ 'Allowlist and denylist for wanchain should be consistent with assets' OK
2022-02-17T12:33:28.8813593Z -  ✓ 'Allowlist and denylist for thundertoken should be consistent with assets' OK
2022-02-17T12:33:28.9077946Z -  ✓ 'Allowlist and denylist for smartchain should be consistent with assets' OK
2022-02-17T12:33:28.9086098Z -  ✓ 'Allowlist and denylist for polygon should be consistent with assets' OK
2022-02-17T12:33:28.9105435Z -  ✓ 'Allowlist and denylist for binance should be consistent with assets' OK
2022-02-17T12:33:28.9123639Z -  ✓ 'Allowlist and denylist for tron should be consistent with assets' OK
2022-02-17T12:33:28.9127860Z -  ✓ 'Allowlist and denylist for terra should be consistent with assets' OK
2022-02-17T12:33:28.9246736Z -  ✓ 'Allowlist and denylist for neo should be consistent with assets' OK
2022-02-17T12:33:28.9248118Z -  ✓ 'Allowlist and denylist for nuls should be consistent with assets' OK
2022-02-17T12:33:28.9248843Z -  ✓ 'Allowlist and denylist for vechain should be consistent with assets' OK
2022-02-17T12:33:28.9249357Z -  ✓ 'Allowlist and denylist for ontology should be consistent with assets' OK
2022-02-17T12:33:28.9249869Z -  ✓ 'Allowlist and denylist for theta should be consistent with assets' OK
2022-02-17T12:33:28.9250371Z -  ✓ 'Allowlist and denylist for eos should be consistent with assets' OK
2022-02-17T12:33:28.9250869Z -  ✓ 'Allowlist and denylist for solana should be consistent with assets' OK
2022-02-17T12:33:28.9251278Z - ✓ Action 'Allowlists' OK, all 19 steps
2022-02-17T12:33:28.9251624Z All consistency checks done, found 0 errors, 0 warnings
2022-02-17T12:33:28.9339790Z npm ERR! code ELIFECYCLE
2022-02-17T12:33:28.9340027Z npm ERR! errno 1
2022-02-17T12:33:28.9367887Z npm ERR! assets@1.0.0 check: `ts-node ./script/entrypoint/check`
2022-02-17T12:33:28.9368330Z npm ERR! Exit status 1
2022-02-17T12:33:28.9368522Z npm ERR! 
2022-02-17T12:33:28.9368817Z npm ERR! Failed at the assets@1.0.0 check script.
2022-02-17T12:33:28.9369282Z npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2022-02-17T12:33:28.9429907Z 
2022-02-17T12:33:28.9430174Z npm ERR! A complete log of this run can be found in:
2022-02-17T12:33:28.9430724Z npm ERR!     /home/runner/.npm/_logs/2022-02-17T12_33_28_936Z-debug.log
2022-02-17T12:33:28.9467998Z ##[error]Process completed with exit code 1.
2022-02-17T12:33:28.9520477Z Post job cleanup.
2022-02-17T12:33:29.0680905Z [command]/usr/bin/git version
2022-02-17T12:33:29.0724376Z git version 2.35.1
2022-02-17T12:33:29.0756568Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2022-02-17T12:33:29.0791046Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2022-02-17T12:33:29.1063569Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2022-02-17T12:33:29.1091763Z http.https://github.com/.extraheader
2022-02-17T12:33:29.1099014Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2022-02-17T12:33:29.1131723Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2022-02-17T12:33:29.1603650Z Cleaning up orphan processes